.github/workflows/tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           # SciPy versions for Python 3.9.
           - case-name: old SciPy
             os: ubuntu-latest
-            python-version: 3.6
+            python-version: "3.6"
```

Comment:
tbh, you probably just want to drop support for Python 3.6.  It goes end-of-life in about a week or something, and supporting 5 Python versions is a bit excessive.

Reply:
Good idea. Let's do that for QuTiP 4.7.